cmake.config.h.in

```diff
@@ -0,0 +1,429 @@
+/* config.h.  Generated from config.h.in by configure.  */
+/* config.h.in.  Generated from configure.ac by autoheader.  */
```

Comment:
If it can't be automatically generated, then the header to the file should be changed.  And probably the `#defines` that are unrelated to supported cmake platforms should be deleted.